articles/nginx-reverse-proxy/index.md

```diff
@@ -0,0 +1,147 @@
+# Deploying Multiple Applications to VM with NGINX as a Reverse Proxy
+
+## Introduction
+Ever wondered how more than one application is being deployed to the same machine, and how traffic is being routed to the corresponding applications?
+Keep reading to know all about it!
+## Goals
+By the end of the article, one would be able to understand
+- What a Reverse Proxy is?
+- What NGINX is?
+- How does it help in managing multiple applications?
```

Comment:
replace it with Nginx or Reverse Proxies

Reply:
thanks and done